drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/widgets/ScenarioGrid.java

```diff
@@ -275,26 +276,43 @@ public void signalTestToolsAboutSelectedHeaderCells() {
         }
     }
 
-    private void signalTestToolsHeaderCellSelected(final ScenarioGridColumn scenarioGridColumn,
-                                                   final GridData.SelectedCell selectedHeaderCell,
-                                                   final int uiColumnIndex) {
-        final EnableTestToolsEvent enableTestToolsEvent = getEnableTestToolsEvent(scenarioGridColumn,
-                                                                                  selectedHeaderCell,
-                                                                                  uiColumnIndex);
-        eventBus.fireEvent(enableTestToolsEvent);
+    protected void signalTestToolsHeaderCellSelected(final ScenarioGridColumn scenarioGridColumn,
```

Comment:
@yesamer 
Please rename one of the two methods - signalTestToolsAboutSelectedHeaderCells signalTestToolsHeaderCellSelected - because it is unclear what should be the difference between them. If, on the other side, they are doing exactly the same thing or - said differently - they are part of the same behavior logic, give them the same name.

Reply:
Done: I called the first method 'signalTestTools' because it manages any type of selected cell. 
For the second, I leaved 'signalTestToolsHeaderCellSelected' because it manages only header cells.

Reply:
@yesamer 
Thx!